modules/quadbin/snowflake/doc/QUADBIN_TOCHILDREN.md

```diff
@@ -0,0 +1,28 @@
+### QUADBIN_TOCHILDREN
+
+{{% bannerNote type="code" %}}
+carto.QUADBIN_TOCHILDREN(quadbin, resolution)
+{{%/ bannerNote %}}
+
+**Description**
+
+Returns an array with the children quadbins of a given quadbin for a specific resolution. A children quadbin is a quadbin of higher level of detail that is contained by the current quadbin. Each quadbin has four children by definition.
+
+* `quadbin`: `INT` quadbin to get the children from.
+* `resolution`: `INT` resolution of the desired children.
+
+**Return type**
+
+`ARRAY<INT>`
```

Comment:
In Snowflake unfortunately we only have the type Array not ARRAY<INT>. I think it's a good idea to specify the type inside the array but in the other docs we are just writing ARRAY. It's a pain in the as also trying to convert elements from the array to other types inside a UDF as we were getting correlated subqueries issues all the time.